src/Utilities/Compiler/Options/SymbolNamesWithValueOption.cs

```diff
@@ -337,7 +338,9 @@ static string GetDeclarationId(ISymbol symbol)
             }
         }
 
+#pragma warning disable CA1024 // Use properties where appropriate
```

Comment:
This diagnostic does not seem to be a build warning. We normally avoid adding source suppressions for IDE-suggestions.